src/main/java/ru/geekbrains/lessons/lesson3/TicTacToe.java

```diff
@@ -0,0 +1,295 @@
+package ru.geekbrains.lessons.lesson3;
+
+import java.sql.SQLOutput;
+import java.util.Locale;
+import java.util.Random;
+import java.util.Scanner;
+
+public class TicTacToe {
+
+
+    private static final char DOT_HUMAN = 'X';
+    private static final char DOT_AI = 'O';
+    private static final char DOT_EMPTY= '.';
+
+    private static char [][] field;
+    private static int fieldSizeX;
+    private static int fieldSizeY;
+    private static int winLineSize =4;
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final Random RANDOM = new Random();
+
+    private static int scoreHuman = 0;
+    private static int scoreAi = 0;
+
+    public static void main(String[] args) {
+
+        while (true) {
+            initField();
+            printField();
+            while (true) {
+                humanTurn();
+                printField();
+                if (gameCheck(DOT_HUMAN, "HUMAN WINS!!!")) {
+                    break;
+                }
+                aiTurn();
+                printField();
+                if (gameCheck(DOT_AI, "AI WINS!!!")) {
+                    break;
+                }
+            }
+            System.out.printf("SCORE IS: HUMAN AI\n            %d   %d\n", scoreHuman,scoreAi);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.